src/utils/compose.js

```diff
@@ -0,0 +1,3 @@
+export default function(...funcs) {
+  return Component => funcs.reduce((comp, func) => func(comp), Component);
+}
```

Comment:
Den gjør at vi kan skrive

```javascript
const finalComponent = compose(
  withState(...),
  connect(...)
)(Component);
```

i stedet for

```javascript
const finalComponent = connect(...)(withState(...)(Component))
```

Er åpen for alternative måter å skrive den på, men jeg tror det er en vanlig implementasjon :)
Noen bruker kanskje `reduceRight` i stedet for `reduce`, men jeg ville at funksjonene skal bli apply-et i rekkefølgen de skrives.